start-mycroft.sh

```diff
@@ -189,8 +162,6 @@ if [[ "${1}" == "restart" ]] || [[ "${_opt}" == "restart" ]] ; then
 fi
 _params=$@
 
-check-dependencies
```

Comment:
Instead of removing this script can you make an environment variable check or something to ensure it's not run for you.

The Picroft install for example relies on this mechanism.